airlock/templates/login.html

```diff
@@ -29,17 +29,16 @@ <h1 class="text-3xl break-words font-bold text-slate-900 mt-2 md:mt-0 md:col-spa
         </span>
       </form>
       <div class="prose prose-sm mb-6">
-        <p>
-          You cannot access Github to login from Level 4. Instead, you
-          must generate a single user token to log in with.
-        </p>
-        <p>
-          To do so, you will need to login with Github on your usual
-          browser, outside Level 4. Then click on the Account drop down
-          on the top right, and click Settings.  Then click the button
-          "Generate Single User Token".  You can then use that token to
-          login on Level 4.
-        </p>
+        <p>To login to Airlock, you need to:</p>
+        <ul>
+          <li>Log off the VPN, or otherwise access the internet (e.g. on your phone).</li>
+          <li>Go to https://jobs.opensafely.org, and login with Github if you are not already.</li>
+          <li>Go to the Accounts drop down in the top right, and click Settings.</li>
+          <li>Click on the Generate Single Use Token button, and remember the 3 words shown.</li>
```

Comment:
personal preference to use a word for this, but... 
```suggestion
          <li>Click on the Generate Single Use Token button, and remember the three words shown.</li>
```